java/vector/src/main/codegen/templates/MapWriters.java

```diff
@@ -242,7 +242,7 @@ public void end() {
               <#assign constructorParams = minor.arrowTypeConstructorParams />
             <#else>
               <#assign constructorParams = [] />
-              <#list minor.typeParams?reverse as typeParam>
+              <#list minor.typeParams as typeParam>
```

Comment:
Why change this?

Reply:
Making param order more sane.. It's driving me nuts we are flipping the ordering twice.